opentelemetry-dynatrace/Cargo.toml

```diff
@@ -0,0 +1,87 @@
+[package]
+name = "opentelemetry-dynatrace"
+version = "0.1.0"
+authors = ["OpenTelemetry Authors <cncf-opentelemetry-contributors@lists.cncf.io>"]
+description = "Dynatrace exporters and propagators for OpenTelemetry"
+homepage = "https://github.com/open-telemetry/opentelemetry-rust/tree/main/opentelemetry-dynatrace"
+repository = "https://github.com/open-telemetry/opentelemetry-rust/tree/main/opentelemetry-dynatrace"
+readme = "README.md"
+categories = [
+    "development-tools::debugging",
+    "development-tools::profiling",
+    "asynchronous",
+]
+keywords = ["opentelemetry", "metrics", "dynatrace"]
+license = "Apache-2.0"
+edition = "2018"
+
+[package.metadata.docs.rs]
+all-features = true
+rustdoc-args = ["--cfg", "docsrs"]
+
+[[test]]
+name = "integration"
+path = "tests/http_test.rs"
+required-features = ["rt-tokio"]
+
+[features]
+default = ["metrics", "rt-tokio", "reqwest-client", "reqwest-rustls"]
+
+metrics = ["opentelemetry/metrics"]
+
+reqwest-client = ["reqwest", "opentelemetry-http/reqwest"]
+reqwest-rustls = ["reqwest/rustls-tls-native-roots"]
+reqwest-blocking-client = ["reqwest/blocking", "opentelemetry-http/reqwest"]
+surf-client = ["surf", "opentelemetry-http/surf"]
+isahc-client = ["isahc", "opentelemetry-http/isahc"]
+
+rt-tokio = ["tokio", "opentelemetry/rt-tokio"]
+rt-async-std = ["async-std", "opentelemetry/rt-async-std"]
+
+wasm = [
+    "base64",
+    "futures-util",
+    "getrandom/js",
+    "js-sys",
+    "pin-project",
+    "wasm-bindgen",
+    "wasm-bindgen-futures",
+    "web-sys",
+]
```

Comment:
I don't think we have a `wasm http client`. Do we still need this feature in the crate?

Reply:
hmm from my understanding, `reqwest` (and other http clients) will do some magic when the compilation target is set to wasm32-unknown-unknown. Sure using the feature flag won't do it, but I expect the target audience to understand this.

https://github.com/seanmonstar/reqwest/issues/288